ui/src/main/webapp/lib/components/Information.js

```diff
@@ -1,21 +1,50 @@
 import React from 'react'
 import muiThemeable from 'material-ui/styles/muiThemeable'
 
-import { informationDiv, informationLabel, informationP, informationListItemP } from './styles.less'
+const styles = ({
+  informationDiv: {
+    textAlign: 'left',
+    fontSize: '16px',
+    lineHeight: '24px',
+    width: '100%',
+    display: 'inline-block',
+    position: 'relative',
+    height: '200ms'
+  },
+  informationLabel: {
+    position: 'absolute',
+    lineHeight: '22px',
+    top: '30px',
+    transform: 'scale(0.75) translate(0px, -28px)',
+    transformOrigin: 'left top 0px'
+  },
+  informationP: {
+    position: 'relative',
+    height: '100%',
+    margin: '28px 0px 7px',
+    whiteSpace: 'nowrap'
+  },
+  informationListItemP: {
```

Comment:
This style is already in the `Information.js` component file. I think we can shorten the name a bit.